projects/hslayers/src/components/add-data/common/public-api.ts

```diff
@@ -1,7 +1,10 @@
 export * from './access-rights.interface';
 export * from './add-data-common.module';
-export * from './add-data-target-position.component';
-export * from './add-data-common-url.component';
+export * from './add-data-target-position/add-data-target-position.component';
+export * from './add-data-common-url/add-data-common-url.component';
 export * from './add-data-wms-layer-highlight-directive';
-export * from './capabilities-error-dialog.component';
-export * from './nested-layers-table.component';
+export * from './capabilities-error-dialog/capabilities-error-dialog.component';
+export * from './nested-layers-table/nested-layers-table.component';
+export * from './common-url-add/common-url-add.component';
+export * from './common-url-show-details/common-url-show-details.component';
```

Comment:
why 'show'? I generally try to avoid verbs in component names since they sound like a method name then.